src/components/v2/V2Project/V2ManageTokensSection/V1ProjectTokensSection/V1ProjectTokensSection.tsx

```diff
@@ -0,0 +1,71 @@
+import { Trans } from '@lingui/macro'
```

Comment:
This UI is the **Your V1 balance**

<img width="560" alt="Screen Shot 2022-07-14 at 2 57 28 PM" src="https://user-images.githubusercontent.com/94939382/179124906-4fe7431f-284a-41e1-8c21-8ba6ae03bbc8.png">